#d/TMI/boards/networkboard.c
ob_data (["short":"@@query_short","long":"@@query_long","short.text":"The network board","long.text":"This is a bulletin board. For information on how to use it, use `help board'.","silent_look":1,"id":({"board","bulletin board",}),"last_location":0,])
messages ({(["id":42,"body":"On Wed Apr 17, Nightmare wrote:> If anyone would be so kind as to direct me to docs regarding upgrading>  from I2 to I3 on a tmi-2 lib mud, I would greatly appreciate it.> > Much Thanks,>  Nightmarecd /adm/daemons/networkmkdir oldmv I2 oldmv services oldmv inet* oldmv ts.c oldmv ns.c oldmv nameserver.c oldmkdir I3[ copy in all the files ] [ don't forget /adm/daemons/network/I3.c ]cd /includemkdir oldmv net old[ copy in all the new files again ]Then, take a look at hte following to merge in with your own versions:/cmds/std/_tell.c, reply.c /cmds/std/_who.c/cmds/std/_mail.c/cmds/std/_finger.c/cmds/std/_mudlist.c/cmds/std/_mudinfo.crm  /cmds/std/_ping.cedit the .h files as appropriate, and restart the mud.Also, take a look at the preload files (/adm/etc/preload and /adm/etc/socket_preload)you might also need to chekc the /adm/obj/mail* files.If you still get errors, mail me and I'll have a look at your mud.It might require an upgrade on your driver for some things, in whichcase some other things might break too. But those should mostlybe fixed here, so you can peek awya :)Leto","time":829771871,"poster":"Leto","title":"Re: Intermud 3",]),(["id":43,"body":"On Wed Apr 17, Domini wrote:> Who is working on this?> according to ftpd.c's headers, dozens of people have worked on it> Has it been fixed?> according to the headers, many many bugs have been fixed> And who took my blanket?> :P> > Dom.","time":829781618,"poster":"Hanzou","title":"Re: ftpd stuffed.",]),(["title":"Re: ftpd stuffed.","poster":"Hideyoshi","time":830187065,"body":"On Wed Apr 17, Domini wrote:> > And who took my blanket?I thought I saw Leto sneaking around the other day carrying a blanket...:P-Hideyoshi","id":45,]),(["title":"Re: ftpd stuffed.","poster":"Leto","time":830194699,"body":"On Mon Apr 22, Hideyoshi wrote:> On Wed Apr 17, Domini wrote:> > > > And who took my blanket?> > I thought I saw Leto sneaking around the other day carrying a blanket...> :P> > -HideyoshiYeah, it's very cold and lonely here in the Netherlands. So i figured,a blanket wouldn't hurt ;)Leto","id":46,]),(["id":47,"body":"On Mon Apr 22, Leto wrote:> On Mon Apr 22, Hideyoshi wrote:> > On Wed Apr 17, Domini wrote:> > > > > > And who took my blanket?> > > > I thought I saw Leto sneaking around the other day carrying a blanket...> > :P> > > > -Hideyoshi> > Yeah, it's very cold and lonely here in the Netherlands. So i figured,> a blanket wouldn't hurt ;)> > LetoIt does when wrapped around an anvil...--OH.","time":830407525,"poster":"Ohara","title":"Re: ftpd stuffed.",]),(["title":"I3 not working at all","poster":"Ciao","time":831366731,"body":"from the mud Lostworlds, (dido.fa.indiana.edu 5555 or 6666) I3isn't working at all... I type mudlist, and it says no muds onI3 network, and if I type tell ciao@Teamaye hiI get:*Bad argument 1 to call_other()Expected: string or array or object Got: 0.Trace written to /log/runtimeand a similar message if I try gwiz or code.","id":48,]),(["title":"Re: I3 not working at all","poster":"Moksha","time":831369957,"body":"On Mon May  6, Ciao wrote:> from the mud Lostworlds, (dido.fa.indiana.edu 5555 or 6666) I3> isn't working at all... I type mudlist, and it says no muds on> I3 network, and if I type tell ciao@Teamaye hi> I get:> *Bad argument 1 to call_other()> Expected: string or array or object Got: 0.> Trace written to /log/runtime> and a similar message if I try gwiz or code.That might be that LostWorlds has never even been hooked up to I3, Ciao.I would know cuz you know me from there.- Aura","id":49,]),(["title":"Re I3","poster":"Ciao","time":831414953,"body":"I knew that.... :)","id":50,]),(["title":"Re: I3 not working at all","poster":"Hanzou","time":831434752,"body":"On Mon May  6, Ciao wrote:> from the mud Lostworlds, (dido.fa.indiana.edu 5555 or 6666) I3> isn't working at all... I type mudlist, and it says no muds on> I3 network, and if I type tell ciao@Teamaye hi> I get:> *Bad argument 1 to call_other()> Expected: string or array or object Got: 0.> Trace written to /log/runtime> and a similar message if I try gwiz or code.'update /adm/daemons/network/I3' when that happens","id":51,]),(["title":"LW's I3","poster":"Farrow","time":831434834,"body":"The problem with LostWorlds has to do with the machine, and not withthe lib.  Cause the same thing happened with other libs that were testedon that machine.  And the reason you get those errors is because the i3 daemon isn't loading.","id":52,]),(["title":"update /adm/daemons/network/I3","poster":"Ciao","time":831474255,"body":">update /adm/daemons/network/I3/adm/daemons/network/I3: *Failed to load file: inherit/socketTrace written to /log/catchError in loading object.Error: *Failed to load file: inherit/socketand in log catch*Failed to load file: inherit/socketObject: /adm/daemons/network/I3 at line 224'cmd_hook' at /std/user#9659 at line 131'cmd_update' at /cmds/object/_update at line 162'do_update' at /cmds/object/_update at line 217'CATCH' at /cmds/object/_update at line 217'create' at /adm/daemons/network/I3 at line 224","id":53,]),(["title":"Re:Router","poster":"Leto","time":831475676,"body":"Don't use 193.174.167.120 9000 but 205.218.122.131 9000 inyour i3.hthe I3 daemon tries to reload itself when it can't get a socket.Leto","id":54,]),(["id":55,"body":"Just as a side note, has anyone else ever tried the I3 from TMI on a SUNOS machine?  Or are we the first?Just pondering how to fix this....Crusader","time":832050562,"poster":"Crus","title":"LW I3",]),})
id_ref 55
